### PR TITLE
feat: add Sentry

### PR DESCRIPTION
**Pull request by bodinsamuel**
Add sentry for error reporting.

### CODE REVIEW

File: src/utils/sentry.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Sentry from '@sentry/node';
+
+import log from './../log.js';
```
Comment (Haroenv) [inline src/utils/sentry.js]:
```suggestion
import log from '../log.js';
```

File: src/index.js [hunk 2]
```diff
@@ -324,7 +326,8 @@ async function watch(stateManager, mainIndex) {
   });
 }
 
-function error(err) {
-  log.error(err);
+async function error(err) {
+  sentry.report(err);
+  await wait(5000);
   process.exit(1); // eslint-disable-line no-process-exit
```
Comment (Haroenv) [inline src/index.js]: if we just rethrow the error, will sentry still catch it, or do we need the timeout?

File: src/utils/sentry.js (hunk 1 above)
Comment (bodinsamuel) [reply src/utils/sentry.js]: I have this habits, not sure why :D

File: src/index.js (hunk 2 above)
Comment (bodinsamuel) [reply src/index.js]: The timeout is needed (I will add a comment) if I don't wait the `process.exit` come too soon and the error don't have time to reach their API (I guess)

**Pull request merged**